test/commands/folders.test.js

```diff
@@ -1627,7 +1627,7 @@ describe('Folders', () => {
 
 				assert.deepEqual(actualContents, manyFilesExpectedContents);
 				assert.equal(ctx.stdout, '');
-			});
+			}).timeout(60000);
```

Comment:
Hopefully, that's the only place. If not, we should consider moving the timeout to config